source/braille.py

```diff
@@ -2367,7 +2367,12 @@ def ackTimeoutResetter(param):
 	@classmethod
 	def func(cls):
 		while True:
-			ctypes.windll.kernel32.SleepEx(winKernel.INFINITE, True)
+			# Ensuring with try except that occasional OS errors with SleepEx
+			# do not break loop
+			try:
+				ctypes.windll.kernel32.SleepEx(winKernel.INFINITE, True)
+			except OSError:
+				log.debug("", exc_info=True)

```

Comment:
Can you provide an example log of what is raised here?

Reply:
> Can you provide an example log of what is raised here?

See https://github.com/nvaccess/nvda/pull/13045#issuecomment-1196674561